presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -69,6 +70,7 @@ public OrcRecordReader(
             long splitOffset,
             long splitLength,
             List<OrcType> types,
+            Map<Integer, Type> prestoTypes,
```

Comment:
Merge this with `includedColumns` since they key set of `prestoTypes` contains the same information.